lib/jxl/splines.cc

```diff
@@ -460,9 +469,12 @@ Status QuantizedSpline::Dequantize(const Spline::Point& starting_point,
   // it gives an indication of the influence of the color values on the area
   // that will need to be rendered.
   uint64_t logcolor = std::max(
-      uint64_t(1),
-      static_cast<uint64_t>(CeilLog2Nonzero(
-          uint64_t(1) + std::max(color[1], std::max(color[0], color[2])))));
+      kOne, static_cast<uint64_t>(CeilLog2Nonzero(
+                kOne + std::max(color[1], std::max(color[0], color[2])))));
```

Comment:
How about:

```suggestion
                kOne + std::max({color[0], color[1], color[2]}))));
```

(since C++11)

Reply:
Done